tests/EnumTest.php

```diff
@@ -126,4 +127,31 @@ function it_can_return_a_collection()
             'visual studio code', 'sublime text 3', 'vim',
         ], $editor->all());
     }
+
+    /** @test */
+    function it_can_return_line_for_an_item()
+    {
+        Lang::shouldReceive('get')->once()->with('enum.language.php')->andReturn('PHP');
+
+        $line = Language::ofType(Language::PHP)->line();
+
+        $this->assertEquals($line, 'PHP');
+    }
+
+    /** @test */
+    function it_can_return_list_for_select()
```

Comment:
Can you create a test for the vendor method as well please? You'll need to create a new enum for testing so one has a vendor and one doesn't.